x-pack/auditbeat/module/system/socket/config.go

```diff
@@ -65,6 +65,9 @@ type Config struct {
 	// EnableIPv6 allows to control IPv6 support. When unset (default) IPv6
 	// will be automatically detected on runtime.
 	EnableIPv6 *bool `config:"socket.enable_ipv6"`
+
+	// DisableKprobe allows to disable specific kprobes by their probe name.
+	DisableKprobe map[string]bool `config:"socket.disable_kprobe"`
```

Comment:
the name and value don't match. imo either:
* `disabled_kprobes` with a list of kprobe names to disable
* `kprobes` with a map and status

Reply:
I made it a slice.